ajax/libs/tracing.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "tracing-js",
+	"filename": "tracing.min.js",
+	"description": "Tracing.js is a library/utility to help you debug and trace javascript code. Tracing.js allows you to set functions of your own to be called before or after target functions in case of debugging get called. It is like events for function invocations.",
+	"license": "MIT",
+	"author": "ebobby",
```

Comment:
Not sure should we use ID or author's name here? cc @cdnjs/intern3 @cdnjs/library-reviewer